CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- Fixed vulnerabilty by upgrading redis package.
```

Comment:
Please clarify this: "fixed redis vulnerability reported by security scanners. Due to the way Apostrophe constructs keys there was no actual vulnerability in practice."

Reply:
(It's completely fine to say there was a real vulnerability when there was a real vulnerability in practice, we are open about that of course, but here there was not.)